### DISCUSSION TITLE
Help - What is the latest version of code-server for Intel?

### DISCUSSION THREAD

**Discussion by jeffmwjr** [coder/code-server#2590] [Q&A]
I checked the releases and I don't see an x86_64 option to download. What is the latest version that I can run and where can I download it from that matches the x86_64 architecture.

Thank you.

**Comment by jeffmwjr** [comment]
Any ideas?

**Reply by code-asher** [reply]
You'll want to use `amd64`. That's equivalent to `x86_64`.
